udp_driver/src/udp_manager_node.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Apex.AI, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <udp_manager_node.hpp>
+#include <iostream>
+#include <memory>
+
+namespace udp_driver
+{
+UdpManagerNode::UdpManagerNode(const rclcpp::NodeOptions & options)
+: Node("udp_manager_node", options)
+{
+  RCLCPP_INFO(this->get_logger(), "Initializing udp send services");
+  RCLCPP_INFO(this->get_logger(), "Initializing udp driver creation services");
+
+  create_udp_driver_ =
+    this->create_service<udp_msgs::srv::UdpSocket>(
+    "create_udp_driver",
+    std::bind(
+      &UdpManagerNode::create_driver_handler,
+      this, std::placeholders::_1, std::placeholders::_2));
+}
+
+void UdpManagerNode::create_driver_handler(
+  const std::shared_ptr<udp_msgs::srv::UdpSocket::Request> request,
+  std::shared_ptr<udp_msgs::srv::UdpSocket::Response> response)
+{
+  RCLCPP_INFO(this->get_logger(), "Creating a socket");
```

Comment:
Doesn't this need to actually create the socket?

Reply:
baby steps! just wanted to get this PR in to socialize the way I was going about it. didnt want to develop this just off in a corner and come here with something that you guys wouldnt want.

Reply:
Sorry, didn't mean to get pushy.

Reply:
No worries! My background is not in software so I know the way I do development probably isn't "the standard" 😅 doing my best to help where I can though

Reply:
Totally fine. Adding the `Draft` or `WIP` tag is the way to go (as I see you've done). Thanks!